distro/run/assembly/resources/run.bat

```diff
@@ -38,6 +38,19 @@ IF "x%JAVA_HOME%" == "x" (
   )
 )
 
+SET EXPECTED_JAVA_VERSION=17
+FOR /f "tokens=3" %%g IN ('JAVA -version 2^>^&1 ^| findstr /i "version"') DO (
+  SET JAVA_VERSION=%%g
+)
+SET JAVA_VERSION=%JAVA_VERSION:"=%
+ECHO Java version is %JAVA_VERSION%
+FOR /f "delims=. tokens=1" %%v in ("%JAVA_VERSION%") do (
+  IF %%v LSS %EXPECTED_JAVA_VERSION% (
+    ECHO You must use at least JDK 17 to start Camunda Platform Run.
+    GOTO :End
```

Comment:
This keeps the window open for me with JDK <17. Without the `PAUSE´ it closes as before.
```suggestion
    PAUSE
    GOTO :End
```